samples/sqs/native-integration/sample_receivingnativemessage_sqs_[6.0,).partial.md

```diff
@@ -0,0 +1,15 @@
+On the receiving end, an NServiceBus endpoint is listening to the queue and has a handler in place to handle messages of type `SomeNativeMessage`.
+
+NOTE: For the message to be successfully deserialized by NServiceBus the sender must include the full name of the message class, including the namespace.
```

Comment:
Isn't full name already a well known thing that "including the namespace" is redundant?

Reply:
It might be helpful for some, but I guess the sample shows it using the full class name so that should also be a hint.